source/SnapDump-Server/SDSnapshotListCall.class.st

```diff
@@ -6,7 +6,7 @@ Class {
 
 { #category : #accessing }
 SDSnapshotListCall class >> path [
-	^ #( '/snapshots' )
+	^ '/snapshots'
 ]
```

Comment:
I think this class is deprecated if someone could confirm I'd remove it (in another pr)